libs/RobotKit/include/interface/RobotController.h

```diff
@@ -14,6 +14,9 @@ class RobotController
   public:
 	virtual ~RobotController() = default;
 
+	virtual void startTimeout() = 0;
+	virtual void stopTimeout()	= 0;
```

Comment:
je pense qu'il faut être beaucoup plus précis concernant les noms des timeouts parce qu'on va en avoir plusieurs pour différentes choses.

```suggestion
	virtual void startGoToSleepTimeout() = 0;
	virtual void stopGoToSleepTimeout()	= 0;
```